.github/workflows/publish-storybook.yml

```diff
@@ -1,29 +0,0 @@
-name: Publish storybook
-
-on:
-  workflow_dispatch:
-  pull_request:
-    types: [labeled]
-
-jobs:
-  publish-storybook:
-    if: "${{ github.event_name == 'workflow_dispatch' || contains(github.event.pull_request.labels.*.name, 'autorelease: tagged') }}"
```

Comment:
previously (before merging repos) we were deploying storybook manually or whenever a release happened. but now it will be deployed automatically on every push.

if we want the old behavior, we can look into it after #875. changesets has a way to run arbitrary steps after a release is made. https://github.com/changesets/action#with-publishing

Reply:
I think it's good to deploy storybook after every merge to main. Sometimes we forget to do that manually. 
That said, I think it's good to have an option to deploy manually in case we need to do that from a branch.